src/main/java/ru/bclib/client/render/BaseSignBlockEntityRenderer.java

```diff
@@ -110,4 +188,10 @@ public static void registerRenderLayer(Block block) {
 	static {
 		defaultLayer = RenderType.entitySolid(new ResourceLocation("textures/entity/signs/oak.png"));
 	}
+
+	public static class SignType extends WoodType {

```

Comment:
What is the purpose of this class existence?

Reply:
I didn't want to create a Mixin/AccessWidener for the `WoodType`-Constructor. The class is used to register custom Sign-Models/Types.

Reply:
Is assign to custom wood type necessary? Signs use custom renderer with their own textures (one per entity, it is not different to texture atlas in terms of performance as one entity = always one draw call). Original system used only block as a key for texture and was very flexible

Reply:
Depends, this extension would allow plugins to register custom 3D-models for signs. But using the Oak-Sign Model would work as well. I could remove the custom sign-model part.

Reply:
Removed custom Sign-Models in 3725ed9